### DISCUSSION TITLE
NeuralProphet code error (TypeError, 'auto_lr_find')

### DISCUSSION THREAD

**Discussion by kzaho** [ourownstory/neural_prophet#1219] [General]
NeuralProphet 0.5.2 requirements says pytorch-lightning>=1.7, although it doesn't work with 2.0.0, it worked for me with 1.8.0 or 1.9.4, so you need to put upper-bound lib condition.

**Comment by marwanmusa** [comment]
yeah, I just got this error today

**Comment by noxan** [comment]
@kzaho @marwanmusa  we resolved this issue with release `0.5.3`
